.github/workflows/ci-unit-tests.yml

```diff
@@ -2,9 +2,14 @@ name: Tests
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
```

Comment:
is there any reason to keep master in this list?

Reply:
Mostly to make it easy to revert if anything goes wrong. Once everything works, I'll remove it.